src/components/Avatar/utils/getInitials/getInitials.ts

```diff
@@ -0,0 +1,20 @@
+const capitalize = (str: string) => str.toUpperCase()
+
+const getFirstChar = (str: string) => str.charAt(0)
+
+const getInitial = (str: string) => {
+  const firstChar = getFirstChar(str)
+  const capitalized = capitalize(firstChar)
+  return capitalized
+}
```

Comment:
💭 I get how breaking these down in this way can make it easier to think through a solution, but I wonder if it's necessary to create an abstraction that literally saves zero space and doesn't increase the understandability of the code (I think the indirection decreases it):

```suggestion
const getInitial = (str: string) => {
  return str.charAt(0).toUpperCase()
}
```

I think there are a few other culprits in this file as well.

I'm super guilty of doing this sort of thing all the time, but it's something I'm trying to get better at recognizing.

Would you agree with this assessment, or am I off-base?

Reply:
No I think you're right. Especially given that they are ultimately just wrapping native methods. I think breaking things down into small functions like this can be helpful when they wrap several lines of code versus a one liner in this case. It's not as if we are testing each individually. We are testing the whole.